### DISCUSSION TITLE
Which status should choose with Broken devices?

### DISCUSSION THREAD

**Discussion by liwenjing93** [snipe/snipe-it#13285] [Q&A]
Hi guys, 
My company requires to enter all broken devices into the asset management software for auditing later. I wonder which status I should choose with those?

Besides, with available devices in stock that haven't been assigned for anyone, should I create an asset and choose status: "Ready to deploy"? the functions confused me a lot. 

Thanks guys in advance!

**Comment by TShiremanLCOG** [comment]
Greetings,

We use a Status Name of Damaged and a Status Type of Undeployable. This makes it so that it cannot be checked out. For those assets with that status, we set the location to where we collect broken and damaged assets. Once reviewed, we then delete.

For available assets, they will have the 'Ready to Deploy' status. When assigned, they will gain a tag that says 'Deployed'.
